tests/test_StatusCodeSender.js

```diff
@@ -60,22 +45,35 @@ describe("A status code sender", function () {
 	});
 });
 
-function expectedErrorForStatusCode(expectedError, errorCode) {
-	let mockSender = generateMockSender(errorCode);
+const expectedDefaultError = (errorCode, payload) => {
+	let mockSender = generateMockSender(errorCode, payload);
+	let statusCodeSender = new StatusCodeSender(mockSender);
+	let request = new Request();
+
+	return statusCodeSender.send(request).then(() => {
+	}, error => {
+		console.log(error.error)
+		expect(error.error).to.be.an.instanceOf(errors.DefaultError);
+		expect(error.error.message).to.be.equal(payload.errors[0].message);
+	})
+}
+
+function expectedErrorForStatusCode(expectedError, errorCode, payload) {
```

Comment:
You can remove the `payload` parameter from this method since we don't pass a payload to this method ever

- [ ] Remove `payload` parameter from method

Reply:
i think its being used in the generateMockSender function call

Reply:
I think we were going to but we aren't anymore. You can see if you look at all of the places we are calling `expectedErrorForStatusCode`